app/src/views/Home.vue

```diff
@@ -1,18 +1,247 @@
+          <b-col class="col"><hr /></b-col>
+          <br />
+        </b-row>
+        <b-row class="justify-content-start">
+          <b-col
+            sm="3"
+            v-for="item in data.filter((x) => x['tag'] == tag)"
+            :key="item.title"
+          >
+            <b-card
+              class="m-1"
+              v-if="item['tag'] == tag"
+              :title="item.note['title']"
+              align="left"
+            >
+              <b-card-text>
+                <b-card-text class="small text-muted">
+                  <b-icon icon="clock-fill"></b-icon>
+                  Last updated
+                  {{ timeSince(new Date(item.note["changed"])) }}
+                  ago</b-card-text
+                >
+              </b-card-text>
+            </b-card>
+          </b-col>
+        </b-row>
+        <br />
+      </div>
+      <!-- #endregion -->
+    </div>
+    <!-- #endregion -->
   </div>
 </template>
 
 <script>
 // @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue'
+import { BIcon } from "bootstrap-vue";
+import "vue-sidebar-menu/dist/vue-sidebar-menu.css";
+import { SidebarMenu } from "vue-sidebar-menu";
 
 export default {
-  name: 'Home',
+  name: "Home",
   components: {
-    HelloWorld
-  }
-}
+    BIcon,
+    SidebarMenu,
+  },
+  data() {
+    return {
+      collapsed: true,
+      menu: [
+        {
+          header: true,
+          title: "SkyMD",
+          hiddenOnCollapse: true,
+        },
+        {
+          href: "/",
+          title: "New Note",
+          icon: {
+            element: "b-icon",
+            attributes: {
+              icon: "file-earmark-plus"
+            }
+          }
+        },
+      ],
+      data: [
+        { note: { title: "Sky MD", changed: 1629328556890 } },
+        { tag: "🔥", note: { title: "Hot takes", changed: 1629346676313 } },
+        { tag: "🔥", note: { title: "🌶", changed: 1629346676313 } },
+        {
+          tag: "PANCAKES",
+          note: {
+            title: "How to Improve Your Pancake Game",
+            changed: 1629328556890,
+          },
+        },
+        {
+          tag: "PANCAKES",
+          note: {
+            title: "I ate pancakes for breakfast today",
+            changed: 1629346808004,
+          },
+        },
+        { tag: "PANCAKES", note: { title: "🥞", changed: 1629346808004 } },
+      ],
+    };
+  },
+  computed: {
+    /**
+     * Array of all existing tags
+     */
+    tags() {
+      if (this.data) {
+        // get tags from data
+        let result = this.data.map((x) => {
+          if (x["tag"]) {
+            return x.tag;
+          }
+        });
+        // filter out the null ones
+        result = result.filter((x) => x != null);
+        //filter out duplicates
+        result = [...new Set(result)];
+        return result;
+      }
+    },
+  },
+  methods: {
+    /**
+     * Nice formatting for human readable dates
+     */
+    timeSince(date) {
```

Comment:
This is probably something that multiple places will use, so worth refactoring into a `utils` file that multiple locations can import.

Reply:
That was going to be my next big pull request. Refactoring to vuex store file and removing fake data